airbyte-server/src/main/java/io/airbyte/server/handlers/JobHistoryHandler.java

```diff
@@ -46,16 +51,29 @@ public JobHistoryHandler(JobPersistence jobPersistence) {
     this.jobPersistence = jobPersistence;
   }
 
+  @SuppressWarnings("UnstableApiUsage")
   public JobReadList listJobsFor(JobListRequestBody request) throws IOException {
-    final JobConfig.ConfigType configType = Enums.convertTo(request.getConfigType(), JobConfig.ConfigType.class);
-    final String configId = request.getConfigId();
+    Preconditions.checkNotNull(request.getConfigTypes(), "configType cannot be null.");
```

Comment:
Why is it optional in the spec if it is mandatory here?

Reply:
fixed.